qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -110,7 +110,7 @@
         - others:
             variants:
                 - test_default_model:
-                    no RHEL.9
+                    only RHEL.6, RHEL.7, RHEL.8, Win10, Win2012, Win2016, Win2019
```

Comment:
Not need to consider very old EOL guests.

Reply:
> Not need to consider very old EOL guests.

@PaulYuuu  Updated, thanks.